test/helpers/replacer-async.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+var inherits = require('util').inherits;
+var path = require('path');
+var Promise = require('rsvp').Promise;
+var Filter = require('../../');
+var minimatch = require('minimatch');
+
+module.exports = ReplaceAsyncFilter;
+function ReplaceAsyncFilter(inputTree, _options) {
```

Comment:
I updated these to use ES6, and their respective tests to use `broccoli-test-helper`, which works with ES6